Product/Production/Gateway/ExchangeManagement/src/main/webapp/WEB-INF/glassfish-web.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need this file? I thought we were not supporting Glassfish anymore.

Reply:
need approval to removed glassfish